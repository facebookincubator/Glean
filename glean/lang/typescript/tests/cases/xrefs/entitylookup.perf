{
  "@generated": null,
  "lsif.Definition.1": 30,
  "lsif.DefinitionMoniker.1": 1,
  "lsif.Document.1": 1,
  "lsif.Name.1": 1,
  "src.File.1": 1
}