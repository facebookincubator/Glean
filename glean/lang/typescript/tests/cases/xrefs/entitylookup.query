query: |
  D where
    D = lsif.Definition { file = { file = "glean/lang/typescript/tests/cases/xrefs/example.ts" }, range = { text = "fs" } };
    lsif.DefinitionMoniker { D, { just = { ident = _ } } };
perf: True
transform: [gensort, []]
