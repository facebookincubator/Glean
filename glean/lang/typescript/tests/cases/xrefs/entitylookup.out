[
  "@generated",
  {
    "key": {
      "file": {
        "key": {
          "file": {
            "key": "glean/lang/typescript/tests/cases/xrefs/example.ts"
          },
          "language": 49
        }
      },
      "range": {
        "key": {
          "range": {
            "lineBegin": 9,
            "columnBegin": 8,
            "lineEnd": 9,
            "columnEnd": 9
          },
          "fullRange": {
            "lineBegin": 9,
            "columnBegin": 8,
            "lineEnd": 9,
            "columnEnd": 9
          },
          "text": { "key": "fs" }
        }
      }
    }
  }
]